swarm/CHANGELOG.md

```diff
@@ -2,7 +2,10 @@
 
 - Add `ConnectionEvent::{is_outbound,is_inbound}`. See [PR 3625].
 
+- Rename `NetworkBehaviour::OutEvent` to `NetworkBehaviour::ToSwarm`, `ConnectionHandler::InEvent` to `ConnectionHandler::FromSwarm`, `NetworkBehaviour::OutEvent` to `NetworkBehaviour::ToBehaviour`. See [PR 3848].
```

Comment:
```suggestion
- Rename `NetworkBehaviour::OutEvent` to `NetworkBehaviour::ToSwarm`, `ConnectionHandler::InEvent` to `ConnectionHandler::FromBehaviour`, `ConnectionHandler::OutEvent` to `ConnectionHandler::ToBehaviour`. See [PR 3848].
```
Typo and one of the renames should be different, see other comment!

Reply:
@thomaseizinger Done